includes/database/engine/class-query.php

```diff
@@ -2202,46 +2298,29 @@ protected function delete_item_meta( $item_id = 0, $meta_key = '', $meta_value =
 	/**
 	 * Get registered meta data keys
 	 *
-	 * This is a copy of get_registered_meta_keys() for WordPress versions lower
-	 * than 4.6.0.
-	 *
-	 * @since 3.0
+	 * @since 1.0.0
 	 *
 	 * @param string $object_subtype The sub-type of meta keys
 	 *
 	 * @return array
 	 */
-	protected function get_registered_meta_keys( $object_subtype = '' ) {
-		global $wp_meta_keys;
+	private function get_registered_meta_keys( $object_subtype = '' ) {
 
 		// Get the object type
 		$object_type = $this->apply_prefix( $this->item_name );
 
-		// Use function if it exists
-		if ( function_exists( 'get_registered_meta_keys' ) ) {
-			return get_registered_meta_keys( $object_type, $object_subtype );
-		}
-
-		// This code is for versions of WordPress older than 4.6.0 and can be
```

Comment:
Just double checking.. I think EDD3 requires 4.6+ so removing this is okay, yes?

Reply:
Yup! Currently requires 4.7.0. 👍